src/stores/transactions.js

```diff
@@ -54,8 +122,43 @@ export const useTransactionsStore = defineStore('transactions', () => {
     pageIndex.value = index
   }
 
-  function setSelectedTxType(txType) {
-    selectedTxType.value = txType
+  function setPageLimit(value) {
+    pageLimit.value = value
+  }
+
+  function formatSlug(scope, type) {
```

Comment:
there is a lot of formatting function. After several attempts this is the minimal version